Gemfile

```diff
@@ -2,7 +2,9 @@
 
 source 'https://rubygems.org'
 
-ruby '2.5.1' if ENV.key?('DYNO')
+if ENV.key?('DYNO')
+  ruby File.read(File.expand_path('.ruby-version', __dir__)).strip
+end
```

Comment:
This does not play nicely with rvm IME, because it uses a regex. At least the last time I checked, that may have changed in the mean time. :)

Reply:
Rude!